### ISSUE TITLE
White lines in God of War 2

### ISSUE THREAD

**Issue by Davidjarod11** [TheRhysWyrill/XBSX2#30]
## Description

In certain scenarios strange white lines appear that move with the character, it happens in all resolutions except native ps2, I suspect it has to do with reflections.
[https://youtu.be/AbUAencGi28](url)
### Expected behavior

That these lines do not appear moving next to the character

### Actual behavior

Lines appear moving next to kratos

### Steps to reproduce the bug

1. Open XBSX2 in commercial mode
2. Run GOW 2 with default settings
3. As you progress through the game these lines may appear

### Console
XBOX SERIES S

**Comment by TheRhysWyrill** [comment]
To me that looks to be an upscaling artifact honestly, do those white lines appear when running at native resolution?

**Comment by Davidjarod11** [comment]
No, in native resolution they don't appear, I found that the lines don't appear either if I use the pcsx2 wiki settings that set Pixel Offset to Special (Texture), but this makes it look like a pixelated sun through walls, I tried more settings but have not been successful.
![GOW2](https://user-images.githubusercontent.com/47721967/191644749-e3233edc-d2bf-4f40-90b1-ab31a8132895.png)

**Comment by Davidjarod11** [comment]
Auto flush seems to work now and fixes the sun

**Comment by d0x360** [comment]
Did GOW2 have a 1080i mode or was that just 3?  I swear one of them also had a hidden progressive mode and another had a hidden 1080i mode but I could be thinking of a different Sony title...

Anyways if it did and you enabled it in game I imagine it would fix the lines.

**Comment by Ramzilla95** [comment]
I can confirm that enabling the hidden 1080i mode removes this upscale artifact. To activate it you must be holding L1, L2, L3, Circle and Square simultaneously when the game boots up (can't be done from a Save State). You know you have done it correctly when the "Sony Computer Entertainment Presents" screen is purple text instead of its normal white. It seems that as long as you create a Save State after entering the code and load it up for future playing, the 1080i mode will still be enabled. If not loading from a Save State that had the 1080i mode enabled, you will have to enter the code from boot every time. With the mode enabled, we gotta change some settings though. 

Open up God of War II's "Game Properties," go to the "Rendering" tab in "Graphics," and enable "Manual Hardware Renderer Fixes." After that, open the "Hardware Fixes" tab and set "Auto Flush" to "Enabled (Sprites Only)" (I am unsure if this is still necessary, but it did not negatively impact visuals and is a default fix enabled for the game). Next open the "Upscaling Fixes" tab and set "Half Pixel Offset" to "Special (Texture)" and check "Align Sprite." Make sure to leave "Round Sprite" off or you will still have those white-line upscale artifacts. 

I just finished the beginning part of the game and have a save right before the first Pegasus and it seems to have done the trick. If I find anything worth mentioning throughout the playthrough, I'll make sure to update this post.

**Comment by d0x360** [comment]
omg i remember using that mode on my tv and i was so proud i could use
progressive scan haha!  I didnt even think to try, excellent find ty.

I also found that going into the compatibility settings for the emulator
and changing the DPI override to ignore the systems settings and use its
own helped with some of the artifacts.

Cant wait until i have the time to build my new system,  7950x3d, 32 gigs
ddr5 6600 CL32! a RTX 4090l 5 m.2 SSD's  because i got the ROG Crosshair
x670e Extreme (insane price but the 2 addon cards offset it quite a bit,
plus 14 layer PCB) and its all going in an 011 Dynamic with a custom loop
dual rad liquid cooling setup and some of those lian li infinity fans and
ill match the RGB to the fans... just a tone of blue but its perfect.

Anyways...  AVX512 + 3dd vcache is why im excited haha.  I did a test bench
stress test on the cpu and amazinly all the best rated cores have vcache.
Although despite this going to be a beast i might upgrade to Ryzen 8000
because its a complete new uarch and its doing big little like intel but
instead of actual limited instruction ecores its just using lower clocked
full x64 cores and it has a new CCX interconnect...like an infinity faabric
2.0 thats going in CPU  and GPU's...its a game changer, especially in RDNA4
which will change minds...finally lol.

Sorry for rambling im sick as a dog and bored out of my mind
----
Subscribe to my YouTube <http://www.youtube.com/d0x360>Channel!
